src/config.ts

```diff
@@ -299,3 +334,12 @@ export function getStatusProviderFromConfig(): BaseStatusProvider {
     }
   }
 }
+
+export function getChangelogPolicyFromConfig(): ChangelogPolicy {
+  const config = getConfiguration() || {};
+
+  return (
+    config.changelogPolicy ??
+    (isAfterEpoch() ? ChangelogPolicy.Auto : ChangelogPolicy.None)
```

Comment:
Let's check with SDK devs that we actually want this change. I believe we still haven't found ideal tooling to ensure changelogs are always kept up-to-date in our development flow, so a lot of repositories would rely on `simple` to error when the changelog is missing.

Reply:
Fair point, I can revert this one